core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -74,15 +78,30 @@
  * <pre>{@code
  * ServerBuilder sb = new ServerBuilder();
  * Server server =
- *      sb.port(8080, SessionProtocol.HTTP) // Add a port to listen
- *      .withDefaultVirtualHost() // Add services to the default virtual host.
- *          .service(...)
- *          .serviceUnder(...)
- *      .and().withVirtualHost("*.foo.com") // Add a another virtual host.
- *          .service(...)
- *          .serviceUnder(...)
- *      .and().build(); // Build a server.
+ *     sb.http(8080) // Add a port to listen
+ *       .withDefaultVirtualHost() // Add services to the default virtual host.
+ *           .service(...)
+ *           .serviceUnder(...)
+ *       .and().withVirtualHost("*.foo.com") // Add a another virtual host.
+ *           .service(...)
+ *           .serviceUnder(...)
+ *       .and().build(); // Build a server.
  * }</pre>
+ *
+ *
+ * <h2 id="no_port_specified">What happens if no HTTP(S) port is specified?</h2>
+ *
+ * <p>When no TCP/IP port number or local address is specified, {@link ServerBuilder} will automatically bind
+ * to a random TCP/IP port assigned by O/S. It will serve HTTPS if you configured TLS (or HTTP otherwise), e.g.
```

Comment:
`by the O/S` (or maybe OS, I'm not used to seeing a slash there)

Reply:
Fixed